components/badge/README.md

```diff
@@ -1,51 +1,23 @@
 # Badges
 
-Badges are a lightweight method of annotating or labeling content. Wrap content in the `.badge` class to add an inline badge. The font-size and line-height of a badge inherit from that of it's parent's properties.
+Badges are a lightweight method of annotating or labeling content. Wrap content in the .badge class to add an inline badge. The font-size and line-height of a badge inherit from that of it’s parent’s properties.
 
-```
-<span class="badge">
-  Badge
-</span>
-```
+### Badge States
 
-Badge States
+You may wish to display more than the single badge type, either to create separation in the importance of various actions or to communicate the state of an action or form badge states make this super easy.
 
-You may wish to display more than the single badge type, either to create separation in the importance of various actions or to communicate the state of an action or form.  badge states make this super easy.
+|State|Purpose|Preview|
+|---	|---	|---
+|Default| |![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-default.png)|
+|Success|Represents something positive.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-success.png)|
+|Primary|Represents something more significant than a default.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-primary.png)|
+|Danger|Represents something in a danger or error state.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-danger.png)|
+|Warning|Represents something that we want to warn the user about but not quite extreme as a danger state.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-warning.png)|
+|Outline|Outline badges for when we want the density of the badge to be lighter e.g. when next to data in a table cell|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-outline.png)|
+|With Icon|If need be we can add icons and tooltips to add further context.|![Preview](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/components/badge/assets/badge-icon.png)|
 
-```
-<!-- badge: Default -->
-<span class="badge">
-  Default
-</span>
+### Do & Don't
 
-<!-- badge: Success -->
-<span class="badge badge-success">
-  Success
-</span>
-
-<!-- badge: Info -->
-<span class="badge badge-info">
-  Info
-</span>
-
-<!-- badge: Warning -->
-<span class="badge badge-warning">
-  Warning
-</span>
-
-<!-- badge: Danger -->
-<span class="badge badge-danger">
-  Danger
-</span>
-```
-
-# Rounded Badge
-
-Add the class `.badge-rounded` to any `.badge` element to display it with rounded caps.
-
-```
-<!-- badge: Default -->
-<span class="badge badge-rounded">
-  Default
-</span>
-```
+|Do|Don't|
+|---	|---	|
+|Use rounded rectangles.|Use fully rounded corners (we no longer use this style).|
```

Comment:
I don't get this section. If it is a component giving me a badge, why should I care about if it uses fully rounded corners? I just take it and place it somewhere, right?

Reply:
This also seems like a CSS description which we do not need anymore since we use a component, right?

Reply:
Yes seems like an old description. I will talk to Lee because this was in the docs.